docs/getting_started.rst

```diff
@@ -235,6 +235,45 @@ and exporting resource.
     :doc:`/api_widgets`
         available widget types and options.
 
+Django Natural Keys
+===================
+
+The ForeignKeyWidget also supports using Django's natural key functions. A
+manager class with the get_by_natural_key function is require for importing
+foreign key relationships by the field model's natural key, and the model must
+have a natural_key function that can be serialized as a JSON list in order to
+export data. 
+
+The primary utility for natural key functionality is to enable exporting data
+that can be imported into other Django environments with different numerical
+primary key sequences. The natural key functionality enables handling more
+complex data than specifying either a single field or the PK. 
+
+The example below illustrates how to create a field on the BookResource that
+imports and exports its author relationships using the natural key functions
+on the Author model and modelmanager. 
+
+::
+
+    from core.models import Author, Book
+    from import_export import resources
+    from import_export.fields import Field
+    from import_export.widgets import ForeignKeyWidget
+
+    class BookResource(resources.ModelResource):
+
+        author = Field(
+            column_name = "author",
+            attribute = "author",
+            widget = ForeignKeyWidget(Author, use_natural_foreign_keys=True)
+        )
+
+        class Meta:
+            model = Book
+
+Read more at `Django Serialization <https://docs.djangoproject.com/en/4.0/topics/serialization>`_
+
```

Comment:
This is great, thanks for adding.  I would say that it's not necessary to show the imports for `BookResource` , but it would be useful to see the `ModelManager` implementation, and the setting of `objects = BookManager`

Reply:
I am not sure we ought to be re-documenting natural keys which are spelled out in the link to Django's docs, but if you're sure it'd add value I can. I can remove whatever imports you think ought not to be there.

Reply:
I was able to condense the author example enough I think it fits in there -- hopefully that works. Let me know if there's anything else I can do.  

I was doing some noodling and I could definitely see other ways to approach serializing the natural keys (following the pattern of using a separator like ManytoManyWidget) for example. I realize when changing core functionality like a widget we want to be very cautious and get it right!